docs/contributor-guide.rst

```diff
@@ -135,3 +135,24 @@ Editor config
 
 - `python-black <https://atom.io/packages/python-black>`_
 - `linter-flake8 <https://atom.io/packages/linter-flake8>`_
+
+Overview of Resource and Collection interaction (from_json and from_data confusion) 
+-----------------------------------------------------------------------------------
+.. image:: resource:collectionRoute.png
```

Comment:
@charlottemoremen could you provide a screenshot of the docs with this change to verify that this renders the images correctly?